x/axelarnet/module.go

```diff
@@ -295,7 +295,7 @@ func (am AppModule) OnAcknowledgementPacket(
 	case *channeltypes.Acknowledgement_Result:
 		return setTransferCompleted(ctx, am.keeper, packet.SourcePort, packet.SourceChannel, packet.Sequence)
 	default:
-		if err := am.rateLimiter.RateLimitPacket(ctx, packet, nexus.Incoming); err != nil {
+		if err := am.rateLimiter.RateLimitPacket(ctx, packet, nexus.Incoming, types.NewIBCPath(packet.GetSourcePort(), packet.GetSourceChannel())); err != nil {
```

Comment:
why use source channel for ack and timeout, the source channel is from the incoming chain no?

Reply:
I am also confused why are all the rate limits applied in this file `Incoming`? What is `Incoming` and what is `Outgoing`?

Reply:
Based on [IBC convention](https://github.com/cosmos/ibc-go/blob/8c8b0e862ed629a85d6806c6ac82352b83d04314/modules/core/04-channel/keeper/timeout.go#L30), source channel corresponds to the IBC channel id to the counterparty chain on current chain.

Note that we're already using the source channel in the previous `case` for the same reason when setting the transfer as completed.

Timeout and AckErrors cause a refund of the sent token, i.e unlock from the escrow address, and so are rate limited as an incoming transfer to Axelar. IBC sends cause a lock on the escrow address, so are considered outgoing from Axelar. Adding comments about this.